_posts/2022-09-15-homebrew-maintainer-projects-update.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Maintainer Projects
+author: SMillerDev
+---
+
+Homebrew’s Project Leadership Committee has green-lit two paid projects by our maintainers this year and since both have hit some milestones recently we’d love to give you, our sponsors and users, an update on their progress.
+
+## API Project
+Since version [3.6.0]({% post_url 2022-09-07-homebrew-3.6.0 %}) of Homebrew, we are now choosing to advertise the `HOMEBREW_INSTALL_FROM_API` environment variable to the general public. A lot of work was done by @Rylan12 to make the experience more stable and the API install method is bound to bring a big performance improvement to those who dare enable it.
+
+The goal of the `HOMEBREW_INSTALL_FROM_API` environment variable is to allow Homebrew users to install formulae and casks without needing to have Homebrew/homebrew-core and/or Homebrew/homebrew-cask tapped. This is intended to make `brew update` much faster and required less often and also to save space on the user’s machine .
```

Comment:
```suggestion
The goal of the `HOMEBREW_INSTALL_FROM_API` environment variable is to allow Homebrew users to install formulae and casks without needing to have Homebrew/homebrew-core and/or Homebrew/homebrew-cask tapped. This is intended to make `brew update` much faster and also to save space on the user’s machine.
```

I'm not sure the frequency is really affected, other than that running `brew update` is less painful now so it's not as big of a deal to run it

Reply:
@MikeMcQuaid this was your addition.

Reply:
@Rylan12 @SMillerDev The frequency is not yet affected but we will definitely be able to significantly bump the default `HOMEBREW_AUTO_UPDATE_SECS` from 5 minutes once we're not relying on Homebrew/core or cask updates needing to come through as quickly.